contracts/artifacts/Vault.json

```diff
@@ -0,0 +1,1188 @@
+{
+  "_format": "hh-sol-artifact-1",
```

Comment:
I think this file is leftover from rebasing on top of https://github.com/gnosis/gp-v2-services/pull/630 . Because that PR should already have added this artifact.

Reply:
Yes. I thought I had gotten rid of it. Reason its still here is that the filename there is `BalancerV2Vault` and here is just `Vault` I think. Will take care of this.

Reply:
Yes. Removed. Good eye 👁️